src/main/java/com/gl/donate_receive/service/converter/ItemConverter.java

```diff
@@ -6,21 +6,24 @@
 import com.gl.donate_receive.model.ItemType;
 import org.springframework.stereotype.Component;
 
+import java.util.UUID;
+
 @Component
 public class ItemConverter {
 
-	public Item dtoToModel(ItemDto itemDto) {
+	public Item dtoToModel(ItemDto itemDto, UUID ownerId) {
 		return Item.builder()
 			.name(itemDto.getName())
 			.description(itemDto.getDescription())
 			.type(itemDto.getType())
 			.status(getStatus(itemDto))
+			.ownerId(ownerId)
 			.build();
 	}
 
 	private ItemStatus getStatus(ItemDto itemDto) {
 		return itemDto.getType().equals(ItemType.AVAILABLE) ?
```

Comment:
item.getType() in theory should never be null, but as a general rule we compare a constant to something to avoid NPE